### ISSUE TITLE
Add refresh functionality to refetch data

### ISSUE THREAD

**Issue by BattleRattle** [glaubinix/conference-schedule#3]
Currently data is only loaded after page load. We need a refresh button or "pull down to refresh" functionality.

**Comment by BattleRattle** [comment]
Hey @glaubinix, could you please have a look at this solution? The pull-to-refresh didn't really work and was lagging too much. So I decided to create that button / tab.

PS: the image was used from jquery-mobile. It should be included as data-url later, in order to save a request. But we need a build script first :smile:

**Comment by glaubinix** [comment]
Hey, I am not 100% if the solution really works. Maybe we should try the pull-to-refresh again, once we fixed all the other events? I would really like if we could keep the entire width for the days.

Or at least, if we keep the button, we should give the user some feedback. Right now if you get an instant reply, you do not really see the arrow moving and therefore are not sure, if the thing really works

**Comment by glaubinix** [comment]
One more thing to add: The current implementation changes the width of the day tabs even if the plugin is not enabled. Additionally, I dont know if it is only on my MacBook but the button is not aligned with the days and kind of breaks the layout. If we keep the button, then this should be fixed.

**Comment by BattleRattle** [comment]
> One more thing to add: The current implementation changes the width of the day tabs even if the plugin is not enabled.

Yeah, I know. I think, we need a plugin mechanism, which allows us to define additional resources like CSS, images, or maybe plugin-specific configurations. In that way the current refresh plugin could override the tab widths with custom CSS rules, which are only added, when the plugin is enabled.

> Additionally, I dont know if it is only on my MacBook but the button is not aligned with the days and kind of breaks the layout. If we keep the button, then this should be fixed.

Yepp, that's why I created #7 :)
The reason is a different default font-size in different browsers (e.g. desktop vs. mobile). And then the line height is different, too. We need to change all the units from px to em in order to fix this.
PS: I'll try to resolve this one in the next days. But then we would still have the other problem regarding the CSS rules.
